public/index.html

```diff
@@ -28,10 +26,9 @@
     />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;400;500;600&display=swap"
-      rel="stylesheet"
-    />
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;400;500;600&display=swap" rel="preload" as="style" onload="this.rel='stylesheet'">
+    <noscript><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;400;500;600&display=swap"></noscript>
```

Comment:
How come this is wrapped in no script tag?

Reply:
@MinhPhu0304 First line depends on javascript to actually apply the stylesheet. Second line is a fallback if you're browser has javascript disabled.

Not much of a case for us, but I copy pasted it from StackOverflow.